package.json

```diff
@@ -13,13 +13,24 @@
     "tsc:test": "tsc && npm run test --no-cache",
     "build:test": "npm run build && npm run test --no-cache",
     "prebuild": "ts-node scripts/preBuild.ts && tsc",
-    "build": "jest --config jest.build.config.js --no-cache",
+    "build": "jest --config jest.build.config.js --no-cache && npm run playground:build",
     "postbuild": "ts-node scripts/postBuild.ts",
     "prettier": "prettier --write --config prettier.config.js .",
-    "prettier:check": "prettier --config prettier.config.js --check .",
+    "prettier:check": "prettier --config prettier.config.js --check '**/*.{js,jsx,ts,tsx}'",
     "prettier:scripts": "prettier --write ./scripts",
     "lint:js": "eslint . --ext .js,.tsx,.ts",
-    "lint:js:fix": "npm run lint:js --fix"
+    "lint:js:fix": "npm run lint:js --fix",
+    "playground:build": "rm -rf dist/playground && mkdir -p dist/playground/public && npm run playground:build-assets && npm run playground:build-ts && npm run playground:copy-static-assets",
+    "playground:build-assets": "npm run playground:build-assets:img",
+    "playground:build-assets:img": "imagemin src/playground/public/assets/img/* --out-dir=dist/playground/public/img",
+    "playground:build-ts": "tsc && mv dist/playground/public/assets/js dist/playground/public/js",
+    "playground:copy-static-assets": "ts-node src/playground/copyStaticAssets.ts",
+    "playground:start": "node dist/playground/app.js",
+    "playground:watch-dev": "npm-watch playground:start-dev",
+    "playground:start-dev": "node dist/playground/app.js"
+  },
+  "watch": {
+    "playground:start-dev": "dist/public/css/*.css"
```

Comment:
So this only watches build target? 🤔

Reply:
Thanks. These dev commands are not working, removing.